safety-architecture/tools/callgraph-tool/doc/licences/licence_info.md

```diff
@@ -0,0 +1,56 @@
+<!--
+SPDX-FileCopyrightText: 2020 callgraph-tool authors. All rights reserved
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Licence and Copyright information in the callgraph-tool
+
+We are doing our best to keep all the licencing and copyright information in place.
```

Comment:
Typo: licencing ==> licensing

Reply:
Fixed multiple occurrences, renamed dir and file